todo_app/tests/test_integration.py

```diff
@@ -0,0 +1,157 @@
+import pytest, os
+from dotenv import find_dotenv, load_dotenv
+import todo_app.app as app
+from unittest.mock import patch, Mock
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    test_app = app.create_app()
+    with test_app.test_client() as client: 
+        yield client
+
+@patch('requests.get')
```

Comment:
This test errors because you're mocking `requests.get` but your code is calling `requests.request` - e.g. you use
```python
response = requests.request("GET", url, params=query)
```
but the mock is expecting something like
```python
response = requests.get(url, params=query)
```

We should line those up, either by mocking `requests.request` or using `requests.get`, so that the integration test correctly mocks requests

Reply:
Yea, got it. Updated the code. Thanks.